### PR TITLE
LG-2734: vertically align title

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-2734](https://jira.mongodb.org/browse/LG-2734)

Before:
Screen Shot 2023-01-09 at 5 21 20 PM

After:
Screen Shot 2023-01-09 at 5 18 31 PM

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx [hunk 1]
```diff
@@ -143,13 +144,8 @@ const ExpandableCard = ({
...
 
   return (
-    <Card
-      darkMode={darkMode}
-      className={cx(cardStyle(darkMode), className)}
-      id={id}
-      {...rest}
-    >
-      <LeafyGreenProvider darkMode={darkMode}>
+    <LeafyGreenProvider darkMode={darkMode}>
```
Comment (brookescarlett) [inline packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:147]: Why do we wrap this component with the provider?

Comment (shaneeza) [reply packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:147]: So that any component passed as a child can use darkMode.

Comment (brookescarlett) [reply packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:147]: Do we do this with every compound component?

Comment (shaneeza) [reply packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:147]: I think so - at the top of my head, I know we use it for tooltip and modal

**Review by brookescarlett — APPROVED** [review]

File: packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx [hunk 2]
```diff
@@ -16,70 +16,15 @@ import {
...
-   */
-  contentClassName?: string;
-
-  /**
-   * Component children
-   */
-  children?: React.ReactNode;
-}
+} from './ExpandableCard.styles';
+import { ExpandableCardProps } from './ExpandableCard.types';
```
Comment (brookescarlett) [inline packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:27]: do we need to update the index file too?

Comment (shaneeza) [reply packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:27]: It was updated!

File: packages/expandable-card/src/index.ts [hunk 3]
```diff
@@ -1 +1 @@
-export { default } from './ExpandableCard';
+export { default } from './ExpandableCard/ExpandableCard';
```
Comment (shaneeza) [inline packages/expandable-card/src/index.ts:1]: @brookescarlett here

Comment (brookescarlett) [reply packages/expandable-card/src/index.ts:1]: great :)

**Pull request merged**